type-c-service/src/service/ucsi.rs

```diff
@@ -71,6 +76,40 @@ impl<'a> Service<'a> {
         }
     }
 
+    /// Upate the CCI connector change field based on the current pending port
+    fn set_cci_connector_change(&self, state: &mut State, cci: &mut GlobalCci) {
+        if let Some(current_port) = state.pending_ports.front() {
+            // UCSI connector numbers are 1-based
+            cci.set_connector_change(GlobalPortId(current_port.0 + 1));
+        } else {
+            // 0 is used to indicate no pending connector changes
```

Comment:
☢️ sentinel value ☢️

Use `Option` or something to make the sentinality explicit.